packages/components/file-uploader/src/file-uploader.css

```diff
@@ -0,0 +1,421 @@
+/* General/default state................................................ */
+
+:host > .file-upload-wrapper > .file-upload-button,
+:host > .file-card-wrapper > .file-card {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "host" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aexperimental-pseudo-usage)